information-schema/information-schema-runaway-watches.md

```diff
@@ -39,25 +40,28 @@ SELECT * FROM INFORMATION_SCHEMA.RUNAWAY_WATCHES\G
 输出结果如下：
 
 ```sql
+mysql> SELECT * FROM INFORMATION_SCHEMA.RUNAWAY_WATCHES ORDER BY id\G
 *************************** 1. row ***************************
-                 ID: 20003
-RESOURCE_GROUP_NAME: rg2
-         START_TIME: 2023-07-28 13:06:08
-           END_TIME: UNLIMITED
-              WATCH: Similar
-         WATCH_TEXT: 5b7fd445c5756a16f910192ad449c02348656a5e9d2aa61615e6049afbc4a82e
+                 ID: 1
+RESOURCE_GROUP_NAME: default
+         START_TIME: 2024-09-09 03:35:31
+           END_TIME: 2024-09-09 03:45:31
+              WATCH: Exact
+         WATCH_TEXT: SELECT variable_name, variable_value FROM mysql.global_variables
              SOURCE: 127.0.0.1:4000
              ACTION: Kill
+               RULE: ProcessedKeys = 666(10)
 *************************** 2. row ***************************
-                 ID: 16004
-RESOURCE_GROUP_NAME: rg2
-         START_TIME: 2023-07-28 01:45:30
-           END_TIME: UNLIMITED
-              WATCH: Similar
-         WATCH_TEXT: 3d48fca401d8cbb31a9f29adc9c0f9d4be967ca80a34f59c15f73af94e000c84
+                 ID: 2
+RESOURCE_GROUP_NAME: default
+         START_TIME: 2024-09-09 03:36:22
+           END_TIME: 2024-09-09 03:46:22
+              WATCH: Exact
```

Comment:
Better use 2 different type2 for these 2 example record.

Reply:
added